faunadb-java/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -830,9 +949,150 @@ public static Value Multiply(Value term1, Value term2, Value term3, Value term4,
   }
 
   /**
+   * Creates a new Modulo function.
+   *
+   * <p><b>Reference</b>: <a href="https://faunadb.com/documentation/queries#misc_functions">FaunaDB Miscellaneous Functions</a></p>
+   */
+  public static Value Modulo(ImmutableList<Value> terms) {
+    return ObjectV("modulo", ArrayV(terms));
+  }
+
+  /**
+   * Creates a new Modulo function operating on the given terms.
+   */
+  public static Value Modulo(Value term1) {
+    return Modulo(ImmutableList.of(term1));
+  }
+
+  /**
+   * Creates a new Modulo function operating on the given terms.
+   */
+  public static Value Modulo(Value term1, Value term2) {
```

Comment:
there's a reason why I expanded these and I forget off the top of my head

On Tuesday, November 17, 2015, Matt Freels <notifications@github.com> wrote:

> In faunadb-java/src/main/java/com/faunadb/client/query/Language.java
> <https://github.com/faunadb/faunadb-jvm/pull/6#discussion_r45144038>:
>
> > +   */
> > +  public static Value Modulo(ImmutableList<Value> terms) {
> > +    return ObjectV("modulo", ArrayV(terms));
> > +  }
> > +
> > +  /**
> > +   * Creates a new Modulo function operating on the given terms.
> > +   */
> > +  public static Value Modulo(Value term1) {
> > +    return Modulo(ImmutableList.of(term1));
> > +  }
> > +
> > +  /**
> > +   * Creates a new Modulo function operating on the given terms.
> > +   */
> > +  public static Value Modulo(Value term1, Value term2) {
>
> doesn't java support varargs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/faunadb/faunadb-jvm/pull/6/files#r45144038>.
>